ufl/mathfunctions.py

```diff
@@ -82,8 +85,13 @@ class Sqrt(MathFunction):
     __slots__ = ()
 
     def __new__(cls, argument):
-        if isinstance(argument, (ScalarValue, Zero)):
-            return FloatValue(math.sqrt(float(argument)))
+        if isinstance(argument, (IntValue, FloatValue, Zero, int, float)):
```

Comment:
use `numbers.Real` instead of `int`/`float`.

Reply:
:+1: